apps/livreGUI/animationController/AnimationController.cpp

```diff
@@ -18,17 +18,12 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include <livreGUI/animationController/AnimationController.h>
+#include "AnimationController.h"
```

Comment:
Semantically the local include with "" is correct here. The global-scope include should be reserved for 3rd-party and non-project includes imo.